rcaracaus/pledge-vegan-node
# Blog listing title

Steps to test

1. Go to localhost:3000
2. Check to see it has a title "Blogs" at the top of the page in a h1 and on the tab
